### PR TITLE
Updated flagellum momentum to 30

### LINKED ISSUE #1325: Editor cell speed calculation is incorrect
shot
shot2

Probably happened after: https://github.com/Revolutionary-Games/Thrive/pull/1273 was merged

### PR DESCRIPTION
**Pull request by CoreyHendrey**
Updated the flagella momentum to 30 based on @buckly90's suggestion.

Closes #1325

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
Was this playtested already @buckly90 ?
This can be merged once that's done.

**Comment by CoreyHendrey** [comment]
Yeah it was, we can get his stamp of approval here first though if you want.

**Comment by buckly90** [comment]
The change to 30 momentum is perfectly suitable. We have not tried anything higher though, if you would like to do so first.

**Comment by hhyyrylainen** [comment]
I think it would make sense to test multiple different values before accepting the best one.

**Comment by buckly90** [comment]
Did some additional play testing, 50 momentum feels more preferable to me. It's not super fast, especially for eukaryotes but it feels more comfortable  to me than 30 was, personally. Feel free to try it and get your own opinion on it, I might be biased because I like things fast.

File: simulation_parameters/microbe_stage/organelles.json [hunk 1]
```diff
@@ -559,4 +559,4 @@
         "gene": "N",
         "name": "nucleus"
     }
-}
\ No newline at end of file
+}
```
Comment (hhyyrylainen) [inline simulation_parameters/microbe_stage/organelles.json:562]: trailing newline is a problem here

**Comment by CoreyHendrey** [comment]
Yeah, stuck at work. Thought I'd do a cheeky fix in the GitHub editor - but it insists on putting a new line there! Will have to wait till I get home, haha.

**Pull request merged**